common/changes/@microsoft/rush/move-parallelism-parse_2022-09-30-22-57.json

```
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Move parsing of \"--parallelism\" out of execution logic.",
```

```suggestion
      "comment": "",
```

Users won't care about this change.